docs/development/plugins/data/public/kibana-plugin-plugins-data-public.querystringinput.md

```diff
@@ -7,5 +7,5 @@
 <b>Signature:</b>
 
 ```typescript
-QueryStringInput: React.FC<Pick<Props, "prepend" | "query" | "size" | "className" | "placeholder" | "onChange" | "onBlur" | "onSubmit" | "isInvalid" | "indexPatterns" | "dataTestSubj" | "screenTitle" | "disableAutoFocus" | "persistedLog" | "bubbleSubmitEvent" | "languageSwitcherPopoverAnchorPosition" | "onChangeQueryInputFocus">>
+QueryStringInput: React.FC<Pick<Props, "size" | "className" | "placeholder" | "onChange" | "onBlur" | "onSubmit" | "prepend" | "isInvalid" | "query" | "indexPatterns" | "dataTestSubj" | "screenTitle" | "disableAutoFocus" | "persistedLog" | "bubbleSubmitEvent" | "languageSwitcherPopoverAnchorPosition" | "onChangeQueryInputFocus">>
```

Comment:
@restrry I can certainly attend to this but it wasn't clear to me that this would be an ongoing issue. Perhaps I'm missing a change in this PR thats causing this to happen.

Reply:
@mattkime thank you for fixing QueryStringInput in https://github.com/elastic/kibana/pull/78336/
note that `SearchBar` & `FilterBar` components have the same problem